README.md

```diff
@@ -26,6 +26,15 @@ docker-compose up
 
 Press CTRL+C to terminate it.
 
+### Run the background jobs and syncer scripts
+
+In distinct terminals:
+
+```sh
+docker-compose exec api make -f config/docker-dev run-background-jobs
+docker-compose exec api make -f config/docker-dev run-syncer
+```
```

Comment:
Commands ported from https://github.com/c2corg/v6_api/wiki/Development-environment-on-Linux#run-the-application

Not 100% sure though, as I need to run them (or only ``run-syncer``?) to have the tags actually taken into account in ES, but on the other hand after doing so the simple search no longer works in my instance (no more results)!?

Reply:
First answer : `run-syncer` only is not enough

Reply:
Side note, for this point, we need to put them in the wiki: the README must be the simplest possible help to run the dev env.